### PR TITLE
Added Notes Feature

### LINKED ISSUE #63: Allow user to add some description/short notes
**Describe the feature you'd like**
Add a checkbox in NewProfileFragment, if checked allow the user to add some short note/ description/ key points and display it in the details fragment
If not checked, app should work in current state. 

**Screenshots**

<table>
<tr>
<td></td>
<td></td>
</tr>
</table>

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by coder2699**
Fixes #63 
Screenshots for the change:
<table>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

File: app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt [hunk 1]
```diff
@@ -58,6 +58,12 @@ class DetailsFragment : BottomSheetDialogFragment() {
             binding.profileTxt.text = args.name
             binding.str.text = "${setTimeString(args.shr)}:${setTimeString(args.smin)}"
             binding.end.text = "${setTimeString(args.ehr)}:${setTimeString(args.emin)}"
+            binding.profileNote.text = args.notes
+            if (binding.profileNote.text != "") {
```
Comment (rob729) [inline app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt]: Change to binding.profileNote.visibility = if (binding.profileNote.text.isBlank())...

File: app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt [hunk 2]
```diff
@@ -163,6 +175,11 @@ class NewProfileFragment : Fragment() {
             days = selectedDays.fromJson(args.d, type)
             Utils.selectedDays(days, binding.dayPicker)
             binding.userToDoEditText.setText(args.name)
+            if (args.notes != "") {
```
Comment (rob729) [inline app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt]: change to if(!args.notes.isBlank())

File: app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt (hunk 1 above)
Comment (coder2699) [reply app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt]: @rob729 Done

File: app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt (hunk 2 above)
Comment (coder2699) [reply app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt]: Done

File: app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt [hunk 3]
```diff
@@ -58,6 +58,12 @@ class DetailsFragment : BottomSheetDialogFragment() {
             binding.profileTxt.text = args.name
             binding.str.text = "${setTimeString(args.shr)}:${setTimeString(args.smin)}"
             binding.end.text = "${setTimeString(args.ehr)}:${setTimeString(args.emin)}"
+            binding.profileNote.text = args.notes
+            binding.profileNote.visibility = if (binding.profileNote.text != "") {
```
Comment (rob729) [inline app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt]: in the previous comment, I have mentioned to use isBlank() instead of != ""

Comment (coder2699) [reply app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt]: Done

**Review by rob729 — APPROVED** [review]

**Comment by rob729** [comment]
give @coder2699 25 points

**Pull request merged**